docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,364 @@ pip3 install -U gdown
+* Step 2: Install git lfs
 
 ```bash
+sudo apt-get install git-lfs
+```
+
+* Step 3: Download Donkey Car
+
+Downloading Donkey Car and the Tensorflow wheel for jp46 and tf29
+
+```bash
+mkdir projects
+cd projects
 git clone https://github.com/autorope/donkeycar
+git-lfs clone https://github.com/autorope/jetson
 cd donkeycar
 git checkout main
-pip3 install -e .[nano]
 ```
 
-* Alternatively, to get the latest stable release
+* Step 4: Create the python env and install donkey
+
+Also install tensorflow in the last step
+
+```bash
+mamba env create -f install/envs/jetson46.yml
+conda activate donkey
+conda update pip
+pip install -e .[nano]
+pip install git+https://github.com/autorope/keras-vis.git
+pip install ../jetson/tensorflow-2.9.3-cp39-cp39-linux_aarch64.whl
+```
+
+* Step 5: Check the TF installation
+
+Run python and verify that tensorflow is version 2.9 and trt is version 8.2.1:
+
+```bash
+python
+>>> import tensorflow as tf
+>>> tf.__version__
+>>> from tensorflow.python.compiler.tensorrt import trt_convert as trt
+>>> trt._check_trt_version_compatibility()
+```
+
+* Step 6: Install OpenCV
+
+OpenCV with GStreamer support is required for the Nano camera. However, the
+OpenCV binaries on PYPI don't have that enabled. Hence, we need to build these
+ourselves.
+
+Install opencv 4.6 + Gstreamer following the Q-Engineering Opencv manual install
+[tutorial](https://qengineering.eu/install-opencv-4.5-on-jetson-nano.html).
+
+Note, the `CMake` command should be executed with the following flags:
+
+```bash
+cmake -D CMAKE_BUILD_TYPE=RELEASE \
+      -D CMAKE_PREFIX_PATH=$/home/<user name>/mambaforge/envs/donkey/bin/python3.9 \
+      -D python=ON \
+      -D BUILD_opencv_python2=OFF \
+      -D BUILD_opencv_python3=ON \
+      -D CMAKE_INSTALL_PREFIX=/home/<user name>/mambaforge/envs/donkey \
+      -D OPENCV_EXTRA_MODULES_PATH=~/opencv_contrib/modules \
+      -D EIGEN_INCLUDE_PATH=/usr/include/eigen3 \
+      -D WITH_OPENCL=OFF \
+      -D WITH_CUDA=ON \
+      -D CUDA_ARCH_BIN=5.3 \
+      -D CUDA_ARCH_PTX="" \
+      -D WITH_CUDNN=ON \
+      -D WITH_CUBLAS=ON \
+      -D ENABLE_FAST_MATH=ON \
+      -D CUDA_FAST_MATH=ON \
+      -D OPENCV_DNN_CUDA=ON \
+      -D ENABLE_NEON=ON \
+      -D WITH_QT=OFF \
+      -D WITH_OPENMP=ON \
+      -D BUILD_TIFF=ON \
+      -D WITH_FFMPEG=ON \
+      -D WITH_GSTREAMER=ON \
+      -D WITH_TBB=ON \
+      -D BUILD_TBB=ON \
+      -D BUILD_TESTS=OFF \
+      -D WITH_EIGEN=ON \
+      -D WITH_V4L=ON \
+      -D WITH_LIBV4L=ON \
+      -D OPENCV_ENABLE_NONFREE=ON \
+      -D INSTALL_C_EXAMPLES=OFF \
+      -D INSTALL_PYTHON_EXAMPLES=OFF \
+      -D PYTHON3_PACKAGES_PATH=/home/<user name>/mambaforge/envs/donkey/lib/python3.9/site-packages \
+      -D PYTHON3_LIBRARIES_PATH=/home/<user name>/mambaforge/envs/donkey/lib \
+      -D OPENCV_GENERATE_PKGCONFIG=ON \
+      -D BUILD_EXAMPLES=OFF ..
+
```

Comment:
`$ make -j4` is missing here.

I think it would be better to leave out this and the following steps in this code block, and leave only the `cmake` command which differs from the Qengineering install steps